CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ string (COMPARE EQUAL "adis16507-1" $ENV{DEVICE_ID} DEVICE_FOUND_ADIS16507_1)
 string (COMPARE EQUAL "adis16507-2" $ENV{DEVICE_ID} DEVICE_FOUND_ADIS16507_2)
 string (COMPARE EQUAL "adis16507-3" $ENV{DEVICE_ID} DEVICE_FOUND_ADIS16507_3)
 
+string (COMPARE EQUAL "adis16545-3" $ENV{DEVICE_ID} DEVICE_FOUND_ADIS16545_3)
```

Comment:
should add support for all devices from adis1654x family. the complete list is:
adis16545-1
adis16545-2
adis16545-3
adis16547-1
adis16547-2
adis16547-3